update.go

```diff
@@ -0,0 +1,112 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/mod/semver"
+)
+
+func NewUpdateCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "update",
+		Usage:       "Handles self-updates of the Stream CLI.",
+		Description: "Handles self-updates of the Stream CLI.",
+		Subcommands: []*cli.Command{
+			selfUpdateCmd(config),
```

Comment:
Do you plan to add more subcommands here? If not I think we can remove it and use `update` only

Reply:
true. (btw, this file is just a draft yet.)

Reply:
made it simpler